modules/s2sTesting.js

```diff
@@ -15,8 +15,11 @@ s2sTesting.globalRand = Math.random(); // if 10% of bidderA and 10% of bidderB s
 
 // load s2sConfig
 config.getConfig('s2sConfig', config => {
-  testing = config.s2sConfig && config.s2sConfig.testing;
-  s2sTesting.calculateBidSources(config.s2sConfig);
+  if (config && config.s2sConfig) {
+    const s2sConfigs = Array.isArray(config.s2sConfig) ? config.s2sConfig : [config.s2sConfig];
+    testing = s2sConfigs.some(i => i.testing);
```

Comment:
so if one s2sconfig is true, all are?

Reply:
Yes, this was from the original PR. Was this incorrect?